assets/javascripts/discourse/initializers/checklist.js

```diff
@@ -19,6 +19,7 @@ export function checklistSyntax($elem, postDecorator) {
   const $boxes = $elem.find(".chcklst-box");
   const postWidget = postDecorator.widget;
   const postModel = postDecorator.getModel();
+  const removeReadonlyClass = () => $boxes.removeClass("readonly");
```

Comment:
We shouldn't add any new jquery code, either remove all jquery of this file if you have the time, or make this specific call using vanilla js

Reply:
I'm also concerned this anonymous function is holding reference to DOM which will make it leak most likely